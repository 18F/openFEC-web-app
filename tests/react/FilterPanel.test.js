const rendered = require('react-test-renderer');
const React = require('react');
const ReactDOM = require('react-dom');
const FilterPanel = require('../../static/js/legal/FilterPanel');

it('renders filter panel', () => {
  const component = rendered.create(
    <FilterPanel id="first-content-0" header="Documents" startOpen={true}>tests</FilterPanel>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
