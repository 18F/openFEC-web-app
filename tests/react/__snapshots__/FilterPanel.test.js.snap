// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders filter panel 1`] = `
<li>
  <button
    aria-controls="first-content-0"
    aria-expanded={true}
    className="accordion__button"
    onClick={[Function]}
  >
    Documents
  </button>
  <div
    className="accordion__content filters-inner"
    hidden={false}
    id="first-content-0"
  >
    tests
  </div>
</li>
`;
